styles.css

```diff
@@ -387,3 +385,269 @@ hr {
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  border: none;
+  padding: 12px;
+}
+
+.contact-section .form textarea {
+  background-color: #fbf8f7;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+  padding: 12px;
+}
+
+.contact-section .form .send {
+  padding: 12px;
+  gap: 10px;
+  width: 131px;
+  height: 48px;
+  background-color: #ff6b00;
+  margin: auto;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.contact-section .form .send:hover {
+  background: rgba(0, 0, 0, 0.3);
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+footer .footer-container {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  justify-content: space-around;
+  margin: 30px auto;
+  width: 170px;
+}
+
+footer .footer-container li a {
+  color: #505f79;
+}
+
+footer .ending-line {
+  width: 134px;
+  height: 5px;
+  background: #0d151e;
+  border-radius: 14px;
+  margin: 0 auto 10px auto;
+}
+
+@media screen and (min-width: 768px) {
+  .navbar .bars {
+    display: none;
+  }
+
+  .links-container {
+    display: flex;
+    flex-direction: row;
+    margin-right: 40px;
+  }
+
+  .links-container ul {
+    background-color: #3e3c3c;
+    display: flex;
+    flex-direction: row;
+    margin-top: 20px;
+    list-style-type: none;
+  }
+
+  .links-container ul li {
+    margin-right: 10px;
+  }
+
+  .links-container ul li a {
+    width: 45px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .links-container .message-container {
+    margin-top: 10px;
+  }
+
+  .header .welcome {
+    margin-left: 70px;
+  }
+
+  .header .job-desc {
+    margin-left: 70px;
+  }
+
+  .header .intro {
+    margin-left: 70px;
+  }
+
+  .header .icons-container {
+    position: absolute;
+    left: 0;
+  }
+
+  .header .icons-container ul {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .header .icons-container ul li {
+    margin-top: 30px;
+  }
+
+  .recent-works {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .recent-works h2 {
+    margin-left: 50px;
+  }
+
+  .recent-works hr {
+    width: 70%;
+    margin-right: 50px;
+  }
+
+  .works .head-work {
+    display: flex;
+    flex-direction: row;
+    margin-left: 50px;
+  }
+
+  .works .head-work .image-container {
+    width: 60%;
+  }
+
+  .works .head-work div {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+  }
+
+  .works .works-section {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    margin-left: 36px;
+  }
```

Comment:
- [ ] kindly consider paying attention to this section so as to meet the requirements
- your design

![ma2](https://user-images.githubusercontent.com/60146030/220933063-70d9da57-93d2-495b-be8d-66e74a142d73.PNG)

- Figma design 👇 
![maf](https://user-images.githubusercontent.com/60146030/220933080-3835f576-6895-4edf-9b0c-b0ad7ab2f795.PNG)
- consider displaying only three cards in a single line as it is in Figma and also put into consideration the skills li to fit into the cards as shown above on the Figma design